Sources/TDS/Utilities/TDS+GeneralRules.swift

```diff
@@ -46,10 +46,10 @@ public typealias Precision = UInt8
 /// SCALE
 public typealias Scale = UInt8
 
-#warning("TODO - GEN_NULL")
-#warning("TODO - CHARBIN_NULL")
-#warning("TODO - FRESERVEDBIT")
-#warning("TODO - FRESERVEDBYTE")
+// TODO: GEN_NULL
+// TODO: CHARBIN_NULL
+// TODO: FRESERVEDBIT
+// TODO: FRESERVEDBYTE
 
```

Comment:
Should we also care about `BIT`( BIT = %b0 / %b1 according to docs)

Reply:
This is mainly used for creating typealiases so that the source code can more easily be mapped to the protocol. Im not sure we could map `BIT` to a native swift type, so I dont think it needs to be handled.